docs/sphinx/source/fsd/index.md

```diff
@@ -0,0 +1,64 @@
+
+Means to incorporate all of these requirements in a portable, extensible, platform neutral, vendor neutral format are provided in the remainder of this Format Specification Document (FSD).
+
+### Status of the Project
+
+The ONSWG maintain both a [website](http://www.opennavsurf.org), which contains the current recommended release of the source code and supporting documentation, and a [Git repository](https://github.com/OpenNavigationSurface/BAG), which contains the source code of the BAG library.
+
+The BAG library is designed to be built from source code, including all component libraries that the BAG uses.  All component libraries are covered under open-source licenses, although not all the licenses are alike. Users should ensure that their use of the BAG library, and its dependencies, is compliant with the terms of the appropriate license before using the component libraries.  The ONS source code library itself is licensed under the [3-Clause BSD License](https://opensource.org/licenses/BSD-3-Clause) and you are free to modify, adapt and otherwise reuse the source code, including the construction of derivative products based on the source code, so long as you do so in accordance with the terms of the 3-Clause BSD license.  However, if a bug is found or there is a way to improve part of the library, it is requested that this be communicated to the ONSWG as described in the [BAG Format Extensions](FSD-Extensions.md) chapter.  Extensions to the library shall not be added without permission of the [BAG Architecture Review Board](FSD-BAGARB.md).  Applications for extensions are also covered in the [BAG Format Extensions](FSD-Extensions.md) chapter.
+
+### History
+
+The idea for what became the Open Navigation Surface Project had been discussed within the US hydrographic community for some time, but first crystallized in late 2003 through comments of M. Paton at IVS3D Ltd., which were adopted by B. Calder at CCOM/JHC with the intent of acting as an independent third-party broker for the development of the library. The ONSWG first met in early 2004, and outlined the requirements for the BAG file structure, the ethos of the project and the basic functionality of the source code library.  A presentation on the structure of the project was made at US Hydro 2005 in March, and development continued until the second ONSWG meeting in mid 2005, when the majority of the first release of the library was pulled together in a little under a week.  A second presentation, including demonstrations of the library linked into SAIC SABER, CARIS HIPS and IVS Fledermaus, was given at Shallow Survey 2005 in September and development of the library continued using e-mail and telephone conferencing until the first Candidate Release on 8 February 2006. Following comments from users and further development, the first official release of the library was made on 8 April 2006.
```

Comment:
It would be useful to list some useful additions/improvements to the library after 2006. For instance, VR and metadata layer.